### DISCUSSION TITLE
I think the readability of the new design is the worst.

### DISCUSSION THREAD

**Discussion by MadeByPP** [GitbookIO/community#554] [Feature Requests]
(Both images below are the screens the viewer sees. Not editor)

From

![image](https://github.com/GitbookIO/community/assets/64509269/3638d9e4-3995-47fd-8c6c-9028bcd66a68)

To

![image](https://github.com/GitbookIO/community/assets/64509269/bb699da6-c822-432e-a1a2-57513d84aed9)

Due to the [update](https://changelog.gitbook.com/march-2024/4-march-huge-improvements-to-your-published-documentation) on March 4th, our document has become a really hard to read document. reached.

The image shows an external link and an internal document link inside a hint box.

In the image before the update, anyone can see at a glance that the bottom link points to an internal document.

However, if the design is changed and someone just throws this image at you without explaining it, it will be completely different.

It has changed indistinguishably the same. This is thought to be one of the factors that reduces readability.

From the reader's perspective, the design after the update will be much more unpredictable and difficult to read.

**Comment by SamyPesse** [comment]
Hey 👋  thank you for the feedback. In your screenshot from the app, you are using both a normal link and a "mention" link, they currently look different, but both could be used for internal or external targets.

We'll look at iterating on the readability of content, both in the editor and in published content, and align the 2 experiences.

**Comment by MadeByPP** [comment]
@SamyPesse Both screens are viewed by document users. The above is before the update and below is after the update.

I'm not talking about the difference with the editor.

The problem is that both the mention link and the general link you mentioned are marked as one after the update, making it difficult to distinguish between the "mention" link and the general link.
